spec/requests/reset_password_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'rails_helper'
+
+describe 'ResetPassword', type: :request do
+  let(:user) { create(:user) }
+
+  it 'get forgot password form' do
+    get '/forgot_password'
+    expect(response).to have_http_status(:success)
+  end
+
+  it 'get reset password form' do
+    get '/reset_password'
+    expect(response).to have_http_status(:success)
+  end
+
+  context 'right params when forgot password' do
+    it 'return success' do
+      post '/auth/password', params: { email: user.email, redirect_url: 'http://localhost:3000/reset_password' }
```

Comment:
Is email going to be sent after requesting this action? If so, please check that email was enqueued